egs/fisher_callhome_spanish/s5/local/callhome_make_trans.pl

```diff
@@ -10,67 +11,62 @@
 open(R, "|sort >$reco") || die "Can't open reco2file_and_channel file $!";
 open(O, ">$tmpdir/callhome.text.1") || die "Can't open text file for writing";
 open(G, ">$tmpdir/callhome_spk2gendertmp") || die "Can't open the speaker to gender map file";
+binmode(O, ":utf8");
 while (<T>) {
 	$file = $_;
 	m:([^/]+)\.txt: || die "Bad filename $_";
 	$call_id = $1;
 	print R "$call_id-A $call_id A\n";
 	print R "$call_id-B $call_id B\n";
 	open(I, "<$file") || die "Opening file $_";
+	binmode(I, ":iso88591");
 	#Now read each line and extract information
 	while (<I>) {
         #136.37 138.10 B: Ah, bueno, mamita.
 		chomp;
 
-		my @stringComponents = split(":");
-        my @timeInfo = split(" ", $stringComponents[0]);
-        $stringComponents[1] =~ s/^\s+|\s+$//g ;
-        my $words = $stringComponents[1];
+		my @stringComponents = split(":", $_, 2);
```

Comment:
I think we usually avoid using tabs in the code? Maybe change all the tabs with 2 spaces?